### DISCUSSION TITLE
Herd for Windows starts a process, and shows nothing...

### DISCUSSION THREAD

**Discussion by wordxpression** [beyondcode/herd-community#544] [Q&A]
I installed Herd for Windows today. After installing nothing seems to happen. When I look with the task manager I see 3 Herd processes, and one Herd HelperProcess, but that's it. 

Any suggestions where to look for a cause for 'not starting' ???

**Comment by frankliniwobi** [comment]
I had this same issue yesterday, you should check the top-facing-icon on the bottom right of your task bar, click on it to open a menu, you would see a herd logo inside that menu, click on the herd logo and you should see your herd dashboard

<img width="103" alt="herd service" src="https://github.com/beyondcode/herd-community/assets/102930821/b52b809b-7069-4575-b1d5-9915061aea90">

Just like in the image above. Hope it helps.
